utils/debug.c

```diff
@@ -376,15 +379,13 @@ void print_diff_count(uint64_t base, uint64_t pair)
 	int sign_idx = (pair < base);
 	int64_t diff = pair - base;
 
-	if (diff == 0) {
-		pr_out("%9s", "+0");
-		return;
-	}
-
 	if (out_color == COLOR_ON) {
 		sc = diff_colors[sign_idx];
 		ec = TERM_COLOR_RESET;
 	}
 
-	pr_out("%s%+9"PRId64"%s", sc, diff, ec);
+	if (diff != 0)
+		pr_out("%s%+9"PRId64"%s", sc, diff, ec);
+	else
+		pr_out("%9s", "+0");
```

Comment:
It seems this function doesn't need the change?

Reply:
I just simplified removing extra return statement.